### PR TITLE
Update GET /networking/ipv6/pools

### PR DESCRIPTION
**Pull request by hzoppetti**
added route_target to ipv6 pool and clarified ipv6 language in
GET /networking/ipv6/pools and
GET /networking/ipv6/ranges

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -7820,7 +7820,9 @@ paths:
       - Networking
       summary: List IPv6 Pools
       description: >
-        Displays the IPv6 pools on your Account.
+        Displays the IPv6 pools on your Account. A pool of IPv6 addresses are routed to all of your
+        Linodes in a single Region. Any Linode on your Account may bring up any address in this
```
Comment (yoshizzle) [inline openapi.yaml]: NB: I could see a benefit of linking the word Region to the LIST Regions endpoint.

File: openapi.yaml [hunk 2]
```diff
@@ -7865,7 +7867,9 @@ paths:
       - Networking
       summary: List IPv6 Ranges
       description: >
-        Displays the IPv6 ranges on your Account.
+        Displays the IPv6 ranges on your Account. A range of IPv6 addresses is routed to a single
+        Linode in a given Region. Your Linode is responsible for routing individual addresses in
```
Comment (yoshizzle) [inline openapi.yaml]: Same as above.

Comment (hzoppetti) [reply openapi.yaml]: Added!

File: openapi.yaml (hunk 1 above)
Comment (hzoppetti) [reply openapi.yaml]: Added!

**Pull request merged**